package.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
We shouldn't need these root-level dependencies. Everything should be in either `client` or `server`. If its giving you trouble, lets talk about it.